subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/screen/Screen.kt

```diff
@@ -5,32 +5,22 @@ package com.avito.android.screen
  * Could be activity or fragment or view based
  *
  * Also used for navigation drawer and dialogs for now
- * todo consider separate interface?
  */
 interface Screen {
 
     /**
      * R.id of root view in hierarchy of a screen
      * Used to determine if screen is presented to user (opened)
+     */
+    val rootId: Int
+
+    /**
+     * Should be a full path to a module containing it's [rootId]
      *
-     * Also used to link "Screen" PageObject and "Screen" (android.view.View) in app code
+     * Used to link "Screen" PageObject and "Screen" (android.view.View) in app code
```

Comment:
"used to" has the opposite meaning: https://dictionary.cambridge.org/grammar/british-grammar/used-to

We can say "Links ..." or "Connects ..."

Reply:
My bad. We can say "It is used to ...".